cmd/virt-api/Dockerfile.mini

```diff
@@ -0,0 +1,64 @@
+#
```

Comment:
Why do you have new .mini Dockerfiles for every component?  This is more for us to maintain. Lets use the Dockerfile's we already have.